pkg/flowaggregator/clickhouseclient/clickhouseclient.go

```diff
@@ -438,7 +438,20 @@ func ConnectClickHouse(config *ClickHouseConfig) (*sql.DB, error) {
 		opt.Compression = &clickhouse.Compression{
 			Method: compression,
 		}
-
+		if tlsEnable { // #nosec G402: ignore insecure options
+			opt.TLS = &tls.Config{
+				InsecureSkipVerify: config.InsecureSkipVerify,
+			}
+			if config.CustomCACert {
+				caCertPool := x509.NewCertPool()
+				successful := caCertPool.AppendCertsFromPEM(config.CACert)
```

Comment:
this type of variable is usually named `ok` by convention (in Golang)

Reply:
fixed.